docs: Custom Applications multiple permissions (#2806)
commercetools/merchant-center-application-kit: website/src/content/api-reference/application-config.mdx

You can have "view-only" or "manage-only" OAuth Scopes and leave the other list field empty, as long as at least one OAuth Scope is specified.

> Alternatively, if at least one additional permission group is configured in `[additionalOAuthScopes](#additionaloauthscopes)`,  both "view-only" or "manage-only" OAuth Scopes list fields can be left empty.

Reviewer: ```suggestion
Alternatively, if at least one additional permission group is configured in [additionalOAuthScopes](#additionaloauthscopes),  both "view-only" or "manage-only" OAuth Scopes list fields can be left empty.
```

Reviewer 2: @kark, can you apply this one as well?

</Info>

## `additionalOAuthScopes`

> The configuration for [additional OAuth Scopes and user permissions](/concepts/oauth-scopes-and-user-permissions#additional-oauth-scopes) extending [oAuthScopes configuration](#oauthscopes).

Reviewer: ```suggestion
<Info>

This feature is available from version `21.21.0` onwards.

</Info>

The optional configuration for defining more granular [OAuth Scopes and user permissions](/concepts/oauth-scopes-and-user-permissions#additional-oauth-scopes).
```

> ## `additionalOAuthScopes.name`

Reviewer: ```suggestion
## `additionalOAuthScopes.*.name`
```

> A name used to build the permission keys based on the default permissions pair.
> 
> The `additionalOAuthScopes.name` value must adhere to the following restrictions:

Reviewer: Let's try to use the same terminology "permission group" to make it easier for users to relate things to each other.

````suggestion
A unique name for the additional permission group.

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
    }
  ]
}
```

The name value must adhere to the following restrictions:
````

* Only lowercase alphabetic characters are allowed.
* Must be between 2 and 64 characters.
* Hyphens are allowed except as leading, trailing, and adjacent characters.

> ## `additionalOAuthScopes.view`
> 
> A list of "view-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `View` permission scoped to the respective permission group.

Reviewer: ```suggestion
## `additionalOAuthScopes.*.view`

A list of "view-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `View<GroupName>` permission.
```

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
      "view": ["view_products"],
      "manage": []
    },
    {
      "name": "merch",
      "view": ["view_categories"],
      "manage": []
    }
  ]
}
```

> ## `additionalOAuthScopes.manage`
> 
> A list of "manage-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `Manage` permission scoped to the respective permission group.

Reviewer: ```suggestion
## `additionalOAuthScopes.*.manage`

A list of "manage-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `Manage<GroupName>` permission.
```

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
    }
  ]
}
```

The name value must adhere to the following restrictions:

* Only lowercase alphabetic characters are allowed.
* Must be between 2 and 64 characters.
* Hyphens are allowed except as leading, trailing, and adjacent characters.

## `additionalOAuthScopes.*.view`

A list of "view-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `View<GroupName>` permission.

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
      "view": ["view_products"],
      "manage": []
    },
    {
      "name": "merch",
      "view": ["view_categories"],
      "manage": []
    }
  ]
}
```

## `additionalOAuthScopes.*.manage`

A list of "manage-only" [OAuth Scopes](https://docs.commercetools.com/api/scopes) required by the Custom Application and associated with the `Manage<GroupName>` permission.

```json
{
  "additionalOAuthScopes": [
    {
      "name": "movies",
      "view": [],
      "manage": ["manage_products"]
    },
    {
      "name": "merch",
      "view": [],
      "manage": ["manage_categories"]
    }
  ]
}
```

> <Info>
> 
> Using `manage_` OAuth Scopes always imply the corresponding `view_` OAuth Scope.
> 
> </Info>

Reviewer 2: ```suggestion
```

We can skip this callout, as it's already shared above.
